### DISCUSSION TITLE
Craft 5 compatibility

### DISCUSSION THREAD

**Discussion by kculmback** [solspace/craft-freeform#1093] [Questions & Answers] [answered]
We're evaluating Freeform as an option for our upcoming site and noticed it doesn't have a Craft 5 compatible version yet, beta or otherwise. Is this on the roadmap to get done or around Craft 5's release?

**Comment by kjmartens** [comment, marked as answer]
Hi @kculmback 

Thank you for your interest in Freeform. We are currently working on making it compatible with Craft 5 and an update for that will be available before the Craft 5 launch. 🙂

**Comment by kculmback** [comment]
@kjmartens that's great to hear, thank you!
